### DISCUSSION TITLE
Slides - box behind text

### DISCUSSION THREAD

**Discussion by cbm214** [elementor/elementor#23060] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Title text or description text doesn't display well over background image.  Buttons display fine but text gets lost.

### Describe the solution you'd like

Able to set a coloured box behind text with levels of transparency. 

### Describe alternatives you've considered

I've found no way to add a background box into slides

**Comment by nematoth** [comment]
I am having the same issue. I am glad it's been raised as a feature request. Should be a standard "out-of-the-box" feature in my opinion. Did you ever find a workaround?
